content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -147,6 +151,14 @@ datasource db {
 }
 ```
 
+In version 2.7.0 and later, you can expand variables using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
+
+```
+# .env
+DATABASE_URL=postgresql://test:test@localhost:5432/test
+DATABASE_URL_WITH_SCHEMA=${DATABASE_URL}?schema=public
+```
```

Comment:
This is not about "Accessing environment variables from the schema" but about the `.env` file. This is _not_ possible in the schema, and not possible with system environment variables. It does not belong in the same headline, but somewhere the `.env` file is explained. If you really want to, you can have a link here "To expand variables and compose them, use the `.env` file."

Reply:
Ah, actually still seems to be there: https://www.prisma.io/docs/concepts/more/environment-variables#expanding-variables